src/main/java/org/folio/consortia/service/impl/SharingPolicyService.java

```diff
@@ -68,6 +74,16 @@ protected String getUrl(SharingPolicyRequest request, HttpMethod httpMethod) {
     return url;
   }
 
+  @Override
+  protected String getSourceValue(SourceValues sourceValue) {
+    return sourceValue.getPolicyValue();
+  }
```

Comment:
getUrl() method that declared in SharingPolicyService should be deleted?

Reply:
thanks, removed as override method and moved to private method